content/docs/2.0/deploy.md

```diff
@@ -53,18 +53,18 @@ If you want to remove KEDA from a cluster you can run one of the following:
 
 ```sh
 helm uninstall -n keda keda
-kubectl delete -f https://raw.githubusercontent.com/kedacore/keda/master/deploy/crds/keda.sh_scaledobjects_crd.yaml
-kubectl delete -f https://raw.githubusercontent.com/kedacore/keda/master/deploy/crds/keda.sh_scaledjobs_crd.yaml
-kubectl delete -f https://raw.githubusercontent.com/kedacore/keda/master/deploy/crds/keda.sh_triggerauthentications_crd.yaml
+kubectl delete -f https://raw.githubusercontent.com/kedacore/keda/v2/config/crd/bases/keda.sh_scaledobjects.yaml
```

Comment:
These seem to be outdated and think these are the ones we need.

Would it make sense if we ship those as part of the release similar to https://github.com/kedacore/keda/releases/download/v2.0.0-rc/keda-2.0.0-rc.yaml?

Reply:
Those CRDs are already part of https://github.com/kedacore/keda/releases/download/v2.0.0-rc/keda-2.0.0-rc.yaml

Reply:
Should we delete this section then?